### PR TITLE
Fix iSCSI kickstart options aren't generated (#1252879)

### PR DESCRIPTION
**Pull request by jkonecny12**
When ISCSI devices are connected in UI, they are not saved to a kickstart. This feature was missing.
Save iSCSI disks to the ksdata. The output kickstart commands are generated automatically from the ksdata.

*Resolves: rhbz#1252879*

Reported-by: Jiri Kortus ``<jikortus@redhat.com>``

### CODE REVIEW

File: pyanaconda/ui/gui/spokes/storage.py [hunk 1]
```diff
@@ -308,6 +308,34 @@ def execute(self):
...
+        # Register iSCSI to kickstart data
+        iscsi_devices = []
+        # Find all selected disks and add all iscsi disks to iscsi_devices list
+        for d in [d for d in getDisks(self.storage.devicetree) if d.name in self.selected_disks]:
+            # Get parents of a multipath devices
+            if isinstance(d, MultipathDevice):
+                for parent_dev in d.parents:
+                    if isinstance(parent_dev, iScsiDiskDevice) and not parent_dev.ibft:
+                        iscsi_devices.append(parent_dev)
+            # Add no-ibft iScsiDiskDevice. IBFT disks are added by firmware so there is
```
Comment (rvykydal) [inline pyanaconda/ui/gui/spokes/storage.py]: To be precise they are not added "by firmware" but "automatically" (by anaconda finding them in iBFT).

**Comment by rvykydal** [comment]
Looks good to me.
We need to filter out also offload iscsi devices but I'll do it in scope of a patch for allowing installing bootloader on offload iscsi devices (using iScsiDiskDevice.offload attribute I will add).

File: pyanaconda/ui/gui/spokes/storage.py (hunk 1 above)
Comment (jkonecny12) [inline pyanaconda/ui/gui/spokes/storage.py]: Ahhhaaa nice to know. I'll change this locally.

**Comment by jkonecny12** [comment]
Pushed

**Pull request closed without merging**